player/lua/select.lua

```diff
@@ -396,8 +396,9 @@ mp.add_key_binding(nil, "show-properties", function ()
 
     -- Don't log errors for renamed and removed properties.
     local msg_level_backup = mp.get_property("msg-level")
-    mp.set_property("msg-level", msg_level_backup == "" and "cplayer=no"
-                                 or msg_level_backup .. ",cplayer=no")
+    local level = "cplayer=no,clipboard=error"
+    mp.set_property("msg-level", msg_level_backup == "" and level
+                                 or msg_level_backup .. "," .. level)
```

Comment:
honestly this is a hack, is there no better way?

Reply:
With #15445 it just won't print the warning.

Reply:
but even the `cplayer=no` part is a hack.
can't we just avoid reading deprecated properties?

Reply:
For sure it is, I guess `options` and `property-list` can be modified to skip renamed and removed options?

Reply:
or add `options/<name>/deprecated` that allows Lua to skip them?

Reply:
This is not so easy to change because `m_config_get_co` unconditionally prints the warning every first time it is called, so even when retrieving the `option-info` of a deprecated property.